db/migrate/20160329215014_add_readonly_to_custom_fields.rb

```diff
@@ -0,0 +1,5 @@
+class AddReadonlyToCustomFields < ActiveRecord::Migration
+  def change
+    add_column :spotlight_custom_fields, :readonly_field, :boolean, default: false
```

Comment:
:ok_hand: figured as much. `readonly_field` is a little clunky, but I guess it'll have to do.